compiler/control/CompileMethod.cpp

```diff
@@ -312,6 +312,10 @@ compileMethodFromDetails(
             compiler.setDebug(createDebugObject(&compiler));
             }
 
+#ifdef VERIFY_IL
```

Comment:
No, it is not. I will switch to `TEST_PROJECT_SPECIFIC`.

I hope it won't be this way for very long - in January, I plan on building out this framework further and will try to refactor current verifiers into using the new system, however I am unsure of how much free time I will have to do this.